src/osg/Texture2D.cpp

```diff
@@ -243,7 +419,6 @@ void Texture2D::apply(State& state) const
     }
     else if (_image.valid() && _image->data())
     {
-        GLExtensions * extensions = state.get<GLExtensions>();
```

Comment:
More code that can go back in now that selectSizedInternalFormat is going back into Texture.cpp